apps/aehttp/src/aehttp_api_handler.erl

```diff
@@ -18,6 +20,12 @@
     validator :: jesse_state:state()
 }).
 
+-ifdef(TEST).
+-define(DEFAULT_HTTP_CACHE_ENABLED, true).
```

Comment:
Why rather not setting a default in `/config/test-lima.config`?

Reply:
(and all other test configs)

Reply:
Rather - if needed for dev1 ... instantiated in integration tests, this shall be set in config of that dev1 ... node.

Reply:
Yeah, I prefer that as well. However I wasn't sure what config file should be used thus is hardcoded constant check in the source.

Changed in 8f2df327a12efbcbd8dcb1f5fc9c2abbd4f66db1